src/phoenix/db/insertion/span.py

```diff
@@ -100,19 +100,30 @@ async def insert_span(
     )
 
     cumulative_error_count = int(span.status_code is SpanStatusCode.ERROR)
-    cumulative_llm_token_count_prompt = cast(
-        int, get_attribute_value(span.attributes, SpanAttributes.LLM_TOKEN_COUNT_PROMPT) or 0
-    )
-    cumulative_llm_token_count_completion = cast(
-        int, get_attribute_value(span.attributes, SpanAttributes.LLM_TOKEN_COUNT_COMPLETION) or 0
-    )
-    llm_token_count_prompt = cast(
-        Optional[int], get_attribute_value(span.attributes, SpanAttributes.LLM_TOKEN_COUNT_PROMPT)
-    )
-    llm_token_count_completion = cast(
-        Optional[int],
-        get_attribute_value(span.attributes, SpanAttributes.LLM_TOKEN_COUNT_COMPLETION),
-    )
+    try:
+        cumulative_llm_token_count_prompt = int(
+            get_attribute_value(span.attributes, SpanAttributes.LLM_TOKEN_COUNT_PROMPT) or 0
+        )
+    except ValueError:
```

Comment:
```suggestion
    except (ValueError, TypeError):
```
e.g.

<img width="417" alt="Screenshot 2025-01-09 at 12 27 30 PM" src="https://github.com/user-attachments/assets/1417e3ce-5a61-447a-b972-50fa0a6a6e55" />